src/styles/_catalog.scss

```
@@ -14,9 +14,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color $transition-time ease, transform $transition-time ease;
```

`&__card:hover` is redundant, you have `&__card`. Use this to your other classes as well

```suggestion
    transition: color $transition-time ease, transform $transition-time ease;
    
    &:hover {
      color: #34568b;
      transform: scale(1.2);
    }
```